sites/feedback/ajax/report-problem-en.html

```diff
@@ -1,78 +1,93 @@
-					<fieldset>
-						<legend>
-							<span class="field-name">Please select all that apply:</span>
-						</legend>
-						<div class="checkbox">
-							<label for="problem1">
-								<input name="problem1" id="problem1" type="checkbox" value="Yes" >A link, button or video is not working
-							</label>
-							<input name="problem1" type="hidden" value="">
-						</div>
-						<div class="checkbox">
-							<label for="problem2">
-								<input name="problem2" id="problem2" type="checkbox" value="Yes" >It has a spelling mistake
-							</label>
-							<input name="problem2" type="hidden" value="">
-						</div>
-						<div class="checkbox">
-							<label for="problem3">
-								<input name="problem3" id="problem3" type="checkbox" value="Yes" >Information is missing
-							</label>
-							<input name="problem3" type="hidden" value="">
-						</div>
-						<div class="checkbox">
-							<label for="problem4">
-								<input name="problem4" id="problem4" type="checkbox" value="Yes" >Information is outdated or wrong
-							</label>
-							<input name="problem4" type="hidden" value="">
-						</div>
-						<div class="checkbox">
-							<label for="problem5">
-								<input name="problem5" id="problem5" type="checkbox" value="Yes" >Login error when trying to access an account
-							</label>
-							<input name="problem5" type="hidden" value="">
-						</div>
-						<div class="checkbox">
-							<label for="problem11">
-								<input name="problem11" id="problem11" type="checkbox" value="Yes" >I can't find what I'm looking for
-							</label>
-							<input name="problem11" type="hidden" value="">
-						</div>
-						<div class="checkbox">
-							<label for="problem12">
-								<input name="problem12" id="problem12" type="checkbox" value="Yes" >Other issue not in this list
-							</label>
-							<input name="problem12" type="hidden" value="">
-						</div>
-					</fieldset>
-					<button type="submit" class="btn btn-primary">Submit</button>
-				</form>
-
+<div class="row row-no-gutters">
+	<div class="col-sm-9 col-md-6 col-lg-5">
+		<details class="brdr-0">
+			<summary class="btn btn-default text-center">Report a problem on this page</summary>
+			<div class="well row">
+				<div class="gc-rprt-prblm">
+					<div class="gc-rprt-prblm-frm gc-rprt-prblm-tggl">
+						<form action="/sites/feedback/feedback-form-destination.html" id="gc-rprt-prblm-form" class="wb-postback"
+							data-wb-postback='{
+								"success": ".success-message",
+								"failure": ".failure-message"}'
+							data-wb-jsonmanager='{
+								"name": "rap",
+								"extractor": [
+									{ "interface": "referer", "path": "externalReferer" },
+									{ "selector": "title", "path": "pageTitle" },
+									{ "interface": "locationHref", "path": "submissionPage" },
+									{ "selector": "html", "attr": "lang", "path": "lang" },
+									{ "selector": "meta[name=\"author\"]", "attr": "content", "path": "pageOwner" },
```

Comment:
Let's use the dublin core metadata corresponding value of `dcterms.creator` as that is the metadata value required according to our [TBS metadata standard in the Appendix C](https://www.tbs-sct.canada.ca/pol/doc-eng.aspx?id=18909)

For the meta value, should we support official alternative according to HTML5 spec? like `dc.creator`, `author`. Except the JSON-LD or/and RDFa equivalent as it might require an improved data extractor.